--- drraw-2.1.1/aq	2004-11-13 21:01:22.000000000 +0100
+++ drraw-2.1.1/drraw.conf	2005-02-13 16:28:51.000000000 +0100
@@ -27,8 +27,8 @@
 # They will be searched recursively for files matching *.rrd (Round Robin
 # Database files) and *.evt (Event files).
 # NOTE: Specify directory paths without any trailing slash
-%datadirs = ('/here/are/some/files'  => '[Label1] ',
-             '/here/are/other/files' => '[Label2] ',
+%datadirs = ('/var/lib/collectd' => '[Label1] ',
+             '/var/lib/rrdtool' => '[Label2] ',
             );
 
 # This function is used to sort *.rrd and *.evt filenames before displaying
@@ -75,8 +75,8 @@
 # These directories MUST EXIST and the user running CGI scripts must have
 # read AND write access to them.  It is entirely safe to delete any content
 # found in the temporary directory.
-$saved_dir = '/somewhere/drraw/saved';
-$tmp_dir = '/somewhere/drraw/tmp';
+$saved_dir = '/home/services/drraw';
+$tmp_dir = '/tmp';
 # By default, critical errors are shown in the produced HTML pages produced
 # and sent to standard error (which web servers typically write to some
 # logfile).  If the following is defined, such errors will be written to
