--- drraw-2.1.1/aq	2004-11-13 21:01:22.000000000 +0100
+++ drraw-2.1.1/drraw.conf	2005-02-13 16:28:51.000000000 +0100
@@ -21,7 +21,7 @@
 # Directories where Round Robin Databases may be found
 # They will be searched recursively for files matching *.rrd (Round Robin
 # Database files) and *.evt (Event files).
-%datadirs = ('/here/are/some/files'  => '[Label1] ',
+%datadirs = ('/home/services/rrd'  => '[Label1] ',
              '/here/are/other/files' => '[Label2] ',
             );
 
@@ -69,8 +69,8 @@
 # These directories MUST EXIST and the user running CGI scripts must have
 # read AND write access to them.  It is entirely safe to delete any content
 # found in the temporary directory.
-$saved_dir = '/somewhere/drraw/saved';
-$tmp_dir = '/somewhere/drraw/tmp';
+$saved_dir = '/home/services/drraw/';
+$tmp_dir = '/tmp';
 # By default, critical errors are shown in the produced HTML pages produced
 # and sent to standard error (which web servers typically write to some
 # logfile).  If the following is defined, such errors will be written to
